experimental/oak_baremetal_runtime/src/framing.rs

```diff
@@ -144,6 +184,7 @@ where
 {
     let mut invocation_handler = InvocationHandler {
         initialization_state: InitializationState::Uninitialized(Some(attestation_behavior)),
+        lookup_data_manager: Arc::new(LookupDataManager::new_empty(StandaloneLogger::default())),
```

Comment:
How is `Arc` implemented on no_std? Does it use some atomic increment x86 instruction?